db/schema.rb

```diff
@@ -10,66 +10,102 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_04_16_183154) do
-
+ActiveRecord::Schema.define(version: 20_210_430_123_359) do
```

Comment:
Not sure if this really matters at all, but it looks like the version is supposed to be a timestamp but the `_` underscores are kinda in the wrong place. Maybe not a big deal though 🤷‍♂️